tests/Foundation/FoundationExpectsModelEventsTest.php

```diff
@@ -106,6 +106,18 @@ public function expects_model_events_can_take_a_string_as_the_event_name()
     }
 
     /** @test */
+    public function expects_events_fires_on_model_events()
+    {
+        EloquentTestModel::created(function () {
+            event(ExampleEvent::class);
+        });
+
+        $this->expectsEvents(ExampleEvent::class);
```

Comment:
Makes no difference because the event will be mocked before the creation and override the ModelEvents.